scripts/erlang-setup.ps1

```diff
@@ -45,8 +43,8 @@ try {
     Install-VCredist
```

Comment:
This can be removed since you install it earlier.

Reply:
I will clean this up